src/test/java/meditracker/medication/MedicationManagerTest.java

```diff
@@ -3,13 +3,24 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
+/**
+ * This test file is to perform tests on the MedicationManager
+ */
+
```

Comment:
There should not be a blank line between the javaDoc and the start of class/method

Reply:
Got it. Resolved in [ac8ad1e](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/240/commits/ac8ad1eafd5b86008ec9a621ca11cfe1d6cbe33f).